components/Main.js

```diff
@@ -81,25 +76,27 @@ const Main = () => {
                     height={180}
                   />
                 ))
-                : <span>Drag photos here</span>
-              }
+              ) : (
+                <span>Drag photos here</span>
+              )}
             </div>
             {imageList.length > 0 && (
               <div className={styles.imageButtons}>
                 <button
                   className={styles.saveButton}
                   onClick={() => {
```

Comment:
The arrow functions in callbacks are bad because they lead to unnecessary renders. You can replace it with a separate handler using  useCallback hook